package.json

```diff
@@ -19,6 +19,7 @@
     "@orchidui/vue": "^0.1.125",
     "@popperjs/core": "^2.11.8",
     "dayjs": "^1.11.10",
+    "echarts": "^5.4.3",
```

Comment:
please add echarts to external build 
<img width="723" alt="Screenshot 2023-12-05 at 03 24 55" src="https://github.com/hit-pay/orchid/assets/46804248/bf474391-06fb-421a-9329-9e24468e694f">